Sources/Commands/UserToolchain.swift

```diff
@@ -25,10 +25,15 @@ struct UserToolchain: Toolchain {
     let sysroot: String?
 
 #if os(OSX)
+    var targetArgs: [String] {
+        return ["-target", "x86_64-apple-macosx10.10"]
+    }
+
     var platformArgs: [String] {
-        return ["-target", "x86_64-apple-macosx10.10", "-sdk", sysroot!]
+        return targetArgs + ["-sdk", sysroot!]
```

Comment:
Why is this necessary?

Clang accepts `-target`, but it isn't really the standard way to pass things to it (the standard way is `-arch x86_64 -mmacosx-version-min=10.10`.